config/config.go

```diff
@@ -57,11 +60,16 @@ func ParseBaseConfig(cfg map[string]string) (Config, error) {
 		return Config{}, err
 	}
 
+	cdcval, err := strconv.ParseBool(cfg[EnableCDC]) //Convert string value
+	if !ok {
+		return Config{}, fmt.Errorf("%v", err)
```

Comment:
`fmt.Errorf("%v", err)` will kinda copy `err` into a new one. We generally do one of things when returning errors:
1. Either return them directly, e.g. `return err`
2. Or we add some context and then return it `fmt.Errorf("cannot parse EnableCDC, input value %v: %w", cdcVal, err)`.